src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+public class Main {
+    private static final int MAX_FIGURE_LENGTH = 7;
+
+    public static void main(String[] args) {
+        FigureSupplier figureSupplier = new FigureSupplier();
+        Figure[] figures = new Figure[MAX_FIGURE_LENGTH];
+
+        for (int i = 0; i < figures.length; i++) {
+            if (i < figures.length / 2) {
+                figures[i] = figureSupplier.getRandomFigure();
+                figures[i].draw();
+            } else {
+                figures[i] = figureSupplier.getDefaultFigure();
+                figures[i].draw();
+            }
+        }
```

Comment:
You should create several random Figures, so you will use a loop. Please don't create a `new FigureSupplier()` inside the loop. It's good that `FigureSupplier` is created only once before the loop starts.